### DISCUSSION TITLE
Cannot make Donation

### DISCUSSION THREAD

**Discussion by Nuge9951** [phw198/OutlookGoogleCalendarSync#1952] [General]
Great software but i cannot make a donation using PayPal it states "Donations to this recipient aren't supported in this country
Return to Paul Woolcock"

Please let me know how to make this donation and remove the splash screen

**Comment by phw198** [comment]
Hi @Nuge9951 and thanks for your interest in donating 😃 Which country are you in? I'm assuming it's one not [supported by PayPal](https://www.paypal.com/uk/webapps/mpp/country-worldwide)?

If you email me directly, I could send you alternative details to transfer me a donation? Thanks again!

**Comment by Nuge9951** [comment]
Hi Paul,

Thanks for your e-mail,  I am based in New Zealand (its clearly on the list) but I have tried several time without luck.

Please send alternate details

Regards
John Ottaway

**Reply by phw198** [reply]
Very strange! Could you email me, please - you can find my address on [my profile page](https://github.com/phw198).
